authorization/resourcetype/scope/repository/resource_type_scope.go

```diff
@@ -196,3 +198,41 @@ func (m *GormResourceTypeScopeRepository) List(ctx context.Context, resourceType
 	}
 	return rows, nil
 }
+
+// ListByResourceTypeAndScope returns all resource type scopes filtered by name and scope.
+func (m *GormResourceTypeScopeRepository) ListByResourceTypeAndScope(ctx context.Context, resourceTypeID uuid.UUID, name string) (*ResourceTypeScope, error) {
```

Comment:
List... suggests multiple results, I would rename this LookupByResourceTypeAndScope instead

Reply:
Done, @sbryzak . https://github.com/fabric8-services/fabric8-auth/pull/347/commits/fc766284c4cab87d941374432e47ea93e7eb3221  Thanks for pointing out !